samza-core/src/main/java/org/apache/samza/coordinator/LeaderElector.java

```diff
@@ -56,4 +56,6 @@
    * @return True, if the caller is the current leader. False, otherwise
    */
   boolean amILeader();
+
+  void close();
```

Comment:
could you provide some documentation on close()? is it true after close() there won't be no more leader?